mirage/utils/create-course-extension-ideas.js

```diff
@@ -9,6 +9,7 @@ export default function createCourseExtensionIdeas(server) {
       slug: courseExtensionIdeaData.slug,
       descriptionMarkdown: courseExtensionIdeaData.description_md,
       votesCount: 0,
+      developmentStatus: courseExtensionIdeaData.development_status,
```

Comment:
Looks like the existing code didn't exactly follow this, but the properties here should be alphabetically ordered too

Reply:
do you want me to re-order all, I did think of it while writing the code but since existing wasn't following it

Reply:
Yep let's reorder -- order doesn't matter here so let's correct the previous mistake